src/text/parsers/timestamp.rs

```diff
@@ -132,38 +128,45 @@ fn timestamp_precision_ymd_hms_fractional(input: &str) -> IResult<&str, TextValu
                 timezone_offset,
             ),
             stop_character,
-        ),
-        |((year, month, day, (hour, minute), second, fractional), offset)| {
-            let builder = Timestamp::with_ymd(year, month, day).with_hms(hour, minute, second);
-            let builder = assign_fractional_seconds(fractional, builder);
-            let timestamp = if let Some(minutes) = offset {
-                builder.build_at_offset(minutes)?
-            } else {
-                builder.build_at_unknown_offset()?
-            };
-            Ok(TextValue::Timestamp(timestamp))
-        },
-    )(input)
+        )(input)?;
+    let builder = Timestamp::with_ymd(year, month, day).with_hms(hour, minute, second);
+    let (_, builder) = assign_fractional_seconds(fractional_text, builder)?;
```

Comment:
🗺️ This is an alternative to the `.1` syntax: pattern match on the tuple and use `_` to throw away the first value (in position `0`).